Flabbergasted-Flamingos/bangazon-commerce-app
# Jdb payment types

# Description
adds payment type list, delete, and add views

## Number of Fixes
3

## Related Ticket(s)
closes #4 
closes #5 

## Problem to Solve
Allows user to list, add, and delete their payment types. 




## Steps to Test Solution
```
git fetch --all
git checkout JDB_payment_types 
```
make sure you have a user signed in, and that there are payment types associated with that user in the db. 

try:
/payment-types to see all payment types.  You should be able to delete a payment there, and then see the updated list without the deleted payment
try:
/payment-types/add to add a payment type.  it should redirect you after submitting the form to the list of payment types, look for the new one you just added.
